### PR TITLE
copy paste strikes again

### PR DESCRIPTION
**Pull request by tesinormed**

### CODE REVIEW

File: viewsgetter.bat [hunk 1]
```diff
@@ -12,14 +12,15 @@ REM this gets all the data that has to do with the user's repos, but we dont rea
 curl "https://api.github.com/users/%user%/repos">%cd%\Repos.txt
 
 REM repo finder extracts the repo names so that we can do something with it
-javac -d out src/com/github/jojo2357/githubviewslogger/*.java
-java -cp $PWD/out/ com.github.jojo2357.githubviewslogger.RepoRefiner
+if not exist "out" mkdir out
```
Comment (jojo2357) [inline viewsgetter.bat]: if not exist "ParsedData\Views" mkdir ParsedData\Views
if not exist "ParsedData\Clones" mkdir ParsedData\Clones

File: viewsgetter.bat [hunk 2]
```diff
@@ -12,14 +12,15 @@ REM this gets all the data that has to do with the user's repos, but we dont rea
...
+java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.RepoRefiner
 
 REM for each repo, we get taffic, and then parse that data
 for /f "delims=" %%x in (Repos.txt) do (
 curl "https://api.github.com/repos/%user%/%%x/traffic/views" -u %user%:%password%>%cd%\%%x.txt
-java com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Views
+java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Views
 curl "https://api.github.com/repos/%user%/%%x/traffic/clones" -u %user%:%password%>%cd%\%%x.txt
-java com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Clones
+java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Clones
```
Comment (jojo2357) [inline viewsgetter.bat]: curl "https://api.github.com/repos/%user%/%%x/traffic/views" -u %user%:%password%>%cd%\%%x.txt
java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Views %directory%
curl "https://api.github.com/repos/%user%/%%x/traffic/clones" -u %user%:%password%>%cd%\%%x.txt
java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Clones %directory%

File: src/com/github/jojo2357/githubviewslogger/GitHubDataParser.java [hunk 3]
```diff
@@ -19,18 +19,18 @@
...
+        if (args.length < 3) {
+            throw new IllegalArgumentException("The project name and clones/views and calling dir is required!");
         }
 
-        final String userDirectory = System.getProperty("user.dir") + "\\";
-        File inputFile = new File(userDirectory + args[0] + ".txt");
-        File outputFile = new File(userDirectory + "ParsedData\\" + args[1] + "\\" + args[0] + ".csv");
+        final String userDirectory = args[2];
         try {
-            Files.createDirectories(Paths.get(URI.create(userDirectory + "ParsedData\\" + args[1])));
```
Comment (jojo2357) [inline src/com/github/jojo2357/githubviewslogger/GitHubDataParser.java]: remove whole try catch...folders managed in scripts now

**Review by jojo2357 — CHANGES_REQUESTED** [review]

File: src/com/github/jojo2357/githubviewslogger/GitHubDataParser.java (hunk 3 above)
Comment (tesinormed) [reply src/com/github/jojo2357/githubviewslogger/GitHubDataParser.java]: aight

File: viewsgetter.bat (hunk 2 above)
Comment (tesinormed) [reply viewsgetter.bat]: must be some rebasing artifacts, will add

Comment (tesinormed) [reply viewsgetter.bat]: did you forget a \ cuz i think %cd%%%x should be %cd%\\%%x

Comment (jojo2357) [reply viewsgetter.bat]: um it disappeared on me. copy paste strikes somehow?

Comment (jojo2357) [reply viewsgetter.bat]: i think i actually updated the file tho it says outdated

**Review by jojo2357 — APPROVED** [review]

**Comment by tesinormed** [comment]
dont merge dont merge

**Review by jojo2357 — APPROVED** [review]

**Pull request merged**